plugins/modules/vmware_guest_disk.py

```diff
@@ -416,6 +421,26 @@ def get_ioandshares_diskconfig(self, disk_spec, disk):
             disk_spec.device.storageIOAllocation = io_disk_spec
         return disk_spec
 
+    def get_sharing(self, disk_type, disk_index):
+        """
+        Get the sharing mode of the virtual disk
+        Args:
+            disk: Virtual disk data object
+
+        Returns:
+            sharing_mode: The sharing mode of the virtual disk
+
+        """
+        sharing = disk.get('sharing')
```

Comment:
`disk` is undefined.

Reply:
Hi @goneri. I noticed this last night when running the tests. I have since fixed, but not pushed it. I have this pull request marked as "draft", I will mark it ready for review and out of draft status when I'm done with the integration tests. Thanks for your initial review!

Reply:
Added the disk parameter and pushed in f3214482d6a202c74491de2286fb1185b1d3849c. Thanks!